recipe/fix.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/include/spdlog/tweakme.h b/include/spdlog/tweakme.h
+index 8fa60e4..339a6ee 100644
+--- a/include/spdlog/tweakme.h
++++ b/include/spdlog/tweakme.h
+@@ -71,7 +71,7 @@
+ // In this case spdlog will try to include <fmt/format.h> so set your -I flag
+ // accordingly.
+ //
+-// #define SPDLOG_FMT_EXTERNAL
++#define SPDLOG_FMT_EXTERNAL
```

Comment:
If someone defines this macro via normal mechanisms this will then cause a macro redefinition warning. This will cause different behaviors for usage of upstream `spdlog` and conda shipped `spdlog` which is undesirable.

Reply:
```suggestion
+#ifndef SPDLOG_FMT_EXTERNAL
+#define SPDLOG_FMT_EXTERNAL
+#endif
```